src/components/accessories/appHeader/AppHeader.tsx

```diff
@@ -10,15 +10,24 @@ const AppHeader: FunctionComponent<TProps> = ({
   userCredentials,
   breadcrumbMap,
 }) => {
+  const appHeader = useRef(null);
   const keys = Object.keys(breadcrumbMap);
   const trailEdgeKey = keys.pop();
+
+  const [isOpen, setIsOpen] = useState(false);
+
   return (
-    <div className="appHeader">
+    <div className={`appHeader ${(isOpen ? "open_menu": "")}`} ref={appHeader}>
       <div className="appHeader__background">
         <div className="appHeader__identifier">
           <div className="appHeader__identifier__logo">
             <img src={logo} alt="Open Hospital" />
           </div>
+          <div className="appHeader__identified__trigger" onClick={() => setIsOpen((!isOpen) ? true : false)}>
```

Comment:
No need to use a ternary here. You can simply pass the opposite value of `isOpen`.

```suggestion
          <div className="appHeader__identified__trigger" onClick={() => setIsOpen(!isOpen)}>
```